compiler/compile/OMRCompilation.cpp

```diff
@@ -326,13 +326,13 @@ OMR::Compilation::Compilation(
    // while the PersistentMethodInfo is allocated during codegen creation
    // Access to this list must be performed with assumptionTableMutex in hand
    //
-   if (!TR::Options::getCmdLineOptions()->getOption(TR_DisableFastAssumptionReclamation))
+   if (!_options->getOption(TR_DisableFastAssumptionReclamation))
       _metadataAssumptionList = new (m->trPersistentMemory()) TR::SentinelRuntimeAssumption();
 #endif
 
    //Random fields must be set before allocating codegen
-   _primaryRandom = new (m->trHeapMemory()) TR_RandomGenerator(options.getRandomSeed());
-   _adhocRandom = new (m->trHeapMemory()) TR_RandomGenerator(options.getRandomSeed());
+   _primaryRandom = new (m->trHeapMemory()) TR_RandomGenerator(_options->getRandomSeed());
```

Comment:
The code was good as it was because `options` is the per compilation Options object

Reply:
yeah I changed it because I saw many more instances of `_options` than `options` so I changed everything to `_options` to avoid confusion. Do you prefer using `options` or just want this part of the code unchanged?

Reply:
Yes, I would prefer to use the parameter `options`